docs/user-guide/application-sonarqube-project-properties.md

```diff
@@ -0,0 +1,54 @@
+# Sonarqube Project Properties for Application
+
+The KubeRocketCI platform simplifies this process by supporting SonarQube integration with predefined configurations. However, specific use cases may require customization. Below is a guide how to set the SonarQube project properties to meet all application requirements.
+
+## Application repo structure
+When application added to the KubeRocketCI, a default `sonar-project.properties` file is automatically generated during the execution process of the build and review pipelines. However, depending on the specific needs of the application, a separate configuration file can be created, placed in the root directory of the application repository, and used for sonarqube code quality during the pipeline:
```

Comment:
>execution process
codebase onboarding process

Reply:
"generated during the execution process of the" - not fixed
we create this file during codebase onboard by codebase-operator